kotest-assertions/kotest-assertions-json/src/commonMain/kotlin/io/kotest/assertions/json/compare.kt

```diff
@@ -155,8 +182,11 @@ enum class TypeCoercion {
     * Types may be coerced. Strings containing numbers will be considered equal to their numbers, and booleans in
     * strings will also be compared.
     *
-    * For example: `"\"11\"".shouldEqualJson("12", compareJsonOptions { typeCoercion = TypeCoercion.Enabled })` will
```

Comment:
`"11".shouldEqualJson("12")` looks like it would always fail, regardless of type coercion? I changed it to `"12".shouldEqualJson("12")`, but that seems like it doesn't demonstrate type coercion properly?

Reply:
@AlexCue987 Do you have an idea what the example should be here? Is what's written correct?